packages/suite/src/views/settings/device/SettingsDevice.tsx

```diff
@@ -85,11 +85,12 @@ export const SettingsDevice = () => {
                     title={<Translation id="TR_SETTINGS_DEVICE_BANNER_TITLE_BOOTLOADER" />}
                     description={
                         <Translation
-                            id={
-                                deviceModel === '1'
-                                    ? 'TR_SETTINGS_DEVICE_BANNER_DESCRIPTION_BOOTLOADER_MODEL_1'
-                                    : 'TR_SETTINGS_DEVICE_BANNER_DESCRIPTION_BOOTLOADER_MODEL_2'
-                            }
+                            id={selectByDeviceModel(deviceModel, {
+                                default:
```

Comment:
Perhaps there is/could be a config/helper for determining touch capabilities?

Reply:
https://www.notion.so/satoshilabs/Model-R-9e5a40f57fc047e3bdd03b4f5404ca4f#74b55ec366bd43cfbb53b1e4319b847d

Reply:
Yeah, I was surprised the touch capability was not already there :D Sure, once it's added things can be changed :)